addons/website_event_online/views/assets.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <template id="assets_frontend" inherit_id="website.assets_frontend" name="Event Assets Frontend">
```

Comment:
Is it ok to have the exact same name as website_event?
I'm guessing it's fine, just making sure.